cloud/vmware/vsphere_guest.py

```diff
@@ -986,7 +988,7 @@ def reconfigure_vm(vsphere_client, vm, module, esxi, resource_pool, cluster_name
         disk_num = 0
         dev_changes = []
         disks_changed = {}
-        for disk in sorted(vm_disk.iterkeys()):
+        for disk in sorted(iterkeys(vm_disk)):
```

Comment:
We can just do this:
``` python

for disk in sorted(vm_disks):
```

There's no need to use iterkeys() from the six module.